Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:latest
```

Comment:
Here I'd prefer to use the LTS by default.

Reply:
the image is now using LTS